core/src/main/java/org/arquillian/smart/testing/scm/git/GitChangeResolver.java

```diff
@@ -45,8 +45,10 @@ public GitChangeResolver() {
             System.getProperty(COMMIT, HEAD));
     }
 
-    public GitChangeResolver(File dir) {
-        this(dir, "HEAD", "HEAD");
+    public GitChangeResolver(File projectDir) {
+        this(projectDir,
+            System.getProperty(PREVIOUS_COMMIT, getPrevCommitDefaultValue()),
+            System.getProperty(COMMIT, HEAD));
```

Comment:
The idea was  that if you only provide a file then we only look at local changes. Why do we need this change?

Reply:
I don't understand your question. 
This is the same constructor as this one: https://github.com/arquillian/smart-testing/pull/121/files#diff-f1255f2030b502bc7deffb7a6797da92R42 with the only difference that you set the path to the project directory. As I have mentioned in the description of the PR - in Che I call ST tool from a different directory than where the root of the project is, which means that this cannot work in my case:
`Paths.get("").toAbsolutePath().toFile()`
But yeah, I can change the referenced constructor to:
```java
    public GitChangeResolver() {
        this(Paths.get("").toAbsolutePath().toFile());
    }
```
to make it more clear

Reply:
Before it `this(dir, "HEAD", "HEAD");` According to the diff, so it is different.

Reply:
Yeah, the `this(dir, "HEAD", "HEAD");` was used only in tests which have been appropriately changed